sklearn/feature_extraction/text.py

```diff
@@ -652,7 +652,7 @@ class HashingVectorizer(TransformerMixin, _VectorizerMixin, BaseEstimator):
         If a callable is passed it is used to extract the sequence of features
         out of the raw, unprocessed input.
 
-        .. versionchanged:: 0.21
+        .. versionadded:: 0.21
```

Comment:
`analyzer` was added long before version 0.21. It should not be marked as added in 0.21.

I'm not sure what this versionchanged intends to refer to, and unless its referent can be clarified, the versioning note should be removed.

Reply:
Ah... I think the versionchanged is with reference to the following content, which should be indented.

Reply:
This was marked as resolved but @jnothman's comment still holds: the `analyzer` param was not added in 0.21, it is much older. It was changed in 0.21 as explained in the next paragraph. The previous annotation was correct.

Reply:
I marked things as resolved as I fixed them, but I coudn't push to the repo. Created a new PR: https://github.com/scikit-learn/scikit-learn/pull/16015